libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -2883,7 +2780,7 @@ static void ProcessTiles(pugi::xml_node Node,
     /* Alloc the type list. Need one additional t_type_desctiptors:
      * 1: empty psuedo-type
      */
-    t_physical_tile_type EMPTY_PHYSICAL_TILE_TYPE = SetupEmptyPhysicalType();
+    t_physical_tile_type EMPTY_PHYSICAL_TILE_TYPE = get_empty_physical_type(std::string("EMPTY"));
```

Comment:
Since you have a default input argument, you can simply write

```
get_empty_physical_type()
```

Reply:
Done